Tests/Engine/DataFeeds/LiveTradingDataFeedTests.cs

```diff
@@ -480,18 +483,17 @@ public void HandlesManyCustomDataSubscriptions()
             Console.WriteLine("Count: " + count);
             Console.WriteLine("Spool up time: " + stopwatch.Elapsed);
 
-            Assert.That(count, Is.GreaterThan(5));
```

Comment:
I'm not sure I understand why this was failing, after https://github.com/QuantConnect/Lean/pull/3040 custom data latency grew to 500ms? It shouldn't off since its also triggering the `OnNewDataAvailable()` event https://github.com/QuantConnect/Lean/pull/3040/files#diff-95f291622834d50427fc1007809439e6R223 , right?